.mdox.validate.yaml

```diff
@@ -1,4 +1,5 @@
 version: 1
+noCache: true
```

Comment:
Again, let's avoid boolean flags. What if we have another cache soon with different type? I believe there should be `cacheType` or something (: e.g `SQlite` and `None`

Reply:
Sure! Will implement this. 🙂